chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx

```diff
@@ -54,8 +52,8 @@ const createMagicCodeSuccessResponse = (signin: string) => {
 export const WebChatContainerStateful = (props: ILiveChatWidgetProps) => {
     const { BasicWebChat } = Components;
     const [state, dispatch]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useChatContextStore();
-    const magicCodeBroadcastChannel = new BroadcastChannel(Constants.magicCodeBroadcastChannel);
-    const magicCodeResponseBroadcastChannel = new BroadcastChannel(Constants.magicCodeResponseBroadcastChannel);
+    const magicCodeBroadcastChannel = new (window as any).BroadcastChannel(Constants.magicCodeBroadcastChannel); // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Why are we creating new Broadcast channel like this instead of using `BroadcastServiceInitialize`? With this change won't there be 2 versions of BroadcastChannel existing at the same time?

Reply:
`BroadcastServiceInitialize` is using the Node version of BroadcastChannel. It cannot be used since `MagicCodeForwarder.html` is using Browser's version of BroadcastChannel. So, the feature in `chat-widget` must use the Browser's version of BroadcastChannel only for the SSO Magic Code feature. Chat widget is still using the Node package version by default